locales/en.json

```diff
@@ -124,9 +124,9 @@
     "global_settings_bad_choice_for_enum": "Bad value for setting {setting:s}, received {received_type:s}, except {expected_type:s}",
     "global_settings_bad_type_for_setting": "Bad type for setting {setting:s}, received {received_type:s}, except {expected_type:s}",
     "global_settings_cant_open_settings": "Failed to open settings file, reason: {reason:s}",
-    "global_settings_cant_serialize_setings": "Failed to serialzed settings data, reason: {reason:s}",
+    "global_settings_cant_serialize_setings": "Failed to serialized settings data, reason: {reason:s}",
```

Comment:
> "Failed to serialize" no ?

Yep.

> btw, there is a typo on the key too : setings.

Nope, [there is two `t`](http://www.linguee.com/english-french/search?source=auto&query=settings).